<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EventPlotPropertiesWidget</class>
 <widget class="QWidget" name="EventPlotPropertiesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Event Plot Properties</string>
  </property>
  <layout class="QVBoxLayout" name="main_layout">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <widget class="QGroupBox" name="x_axis_group">
     <property name="title">
      <string>X-Axis Data</string>
     </property>
     <layout class="QVBoxLayout" name="x_axis_layout">
      <item>
       <layout class="QHBoxLayout" name="x_axis_selection_layout">
        <item>
         <widget class="QLabel" name="x_axis_label">
          <property name="text">
           <string>Data Source:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="x_axis_combo">
          <property name="minimumWidth">
           <number>150</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="x_axis_info_label">
        <property name="text">
         <string>Select a data source for the X-axis</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel { 
color: #555; 
font-size: 11px; 
padding: 8px; 
background-color: #f0f0f0; 
border: 1px solid #ccc; 
border-radius: 4px; 
}</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="interval_settings_group">
     <property name="title">
      <string>Interval Settings</string>
     </property>
     <property name="visible">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="interval_layout">
      <item>
       <widget class="QRadioButton" name="interval_beginning_radio">
        <property name="text">
         <string>Show Beginning of Interval</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="interval_end_radio">
        <property name="text">
         <string>Show End of Interval</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="y_axis_group">
     <property name="title">
      <string>Y-Axis Data</string>
     </property>
     <layout class="QVBoxLayout" name="y_axis_layout">
      <item>
       <widget class="Feature_Table_Widget" name="y_axis_feature_table">
        <property name="maximumHeight">
         <number>200</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="visualization_group">
     <property name="title">
      <string>Visualization Settings</string>
     </property>
     <layout class="QVBoxLayout" name="vis_layout">
      <item>
       <widget class="QLabel" name="x_range_label">
        <property name="text">
         <string>X-Axis Range (Events normalized to 0):</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel { 
font-weight: bold; 
color: #333; 
}</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="negative_range_layout">
        <item>
         <widget class="QLabel" name="negative_range_label">
          <property name="text">
           <string>Negative Range:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="negative_range_spinbox">
          <property name="suffix">
           <string>ms</string>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>30000</number>
          </property>
          <property name="singleStep">
           <number>100</number>
          </property>
          <property name="value">
           <number>30000</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="positive_range_layout">
        <item>
         <widget class="QLabel" name="positive_range_label">
          <property name="text">
           <string>Positive Range:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="positive_range_spinbox">
          <property name="suffix">
           <string>ms</string>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>30000</number>
          </property>
          <property name="singleStep">
           <number>100</number>
          </property>
          <property name="value">
           <number>30000</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="zoom_layout">
        <item>
         <widget class="QLabel" name="zoom_label">
          <property name="text">
           <string>Zoom Level:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="zoom_level_spinbox">
          <property name="suffix">
           <string>x</string>
          </property>
          <property name="minimum">
           <double>0.100000000000000</double>
          </property>
          <property name="maximum">
           <double>10.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_1">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="reset_view_button">
        <property name="text">
         <string>Reset View</string>
        </property>
        <property name="maximumWidth">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="tooltips_checkbox">
        <property name="text">
         <string>Show Tooltips</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Feature_Table_Widget</class>
   <extends>QWidget</extends>
   <header>Feature_Table_Widget/Feature_Table_Widget.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui> 